src/ResourceManager/DataFactories/ChangeLog.md

```diff
@@ -18,7 +18,8 @@
         - Additional information about change #1
 -->
 ## Current Release
-* Fix credential encryption issue that caused no meaningful error for some encryption operations.
+* Fix credential encryption issue that caused no meaningful error for some encryption operations
+* Enable integration runtime to be shared across data factory
```

Comment:
@zhangyd2015 your change should go in the DataFactoryV2 change log, not DataFactories

Reply:
Updated the change log in DataFactoryV2